tests/test_ensembleset.py

```diff
@@ -40,20 +63,17 @@ def test_ensembleset_reek001(tmp="TMP"):
         testdir = os.path.abspath(".")
     ensdir = os.path.join(testdir, "data/testensemble-reek001/")
 
-    # Copy iter-0 to iter-1, creating an identical ensemble
-    # we can load for testing.
-    for realizationdir in glob.glob(ensdir + "/realization-*"):
-        if os.path.exists(realizationdir + "/iter-1"):
-            if os.path.islink(realizationdir + "/iter-1"):
-                os.remove(realizationdir + "/iter-1")
-            else:
-                shutil.rmtree(realizationdir + "/iter-1")
-        os.symlink(realizationdir + "/iter-0", realizationdir + "/iter-1")
+    tmpdir.chdir()
+    symlink_iter(ensdir, "iter-0")
+    symlink_iter(ensdir, "iter-1")
 
-    iter0 = ScratchEnsemble("iter-0", ensdir + "/realization-*/iter-0")
-    iter1 = ScratchEnsemble("iter-1", ensdir + "/realization-*/iter-1")
+    iter0 = ScratchEnsemble("iter-0", str(tmpdir.join("realization-*/iter-0")))
+    iter1 = ScratchEnsemble("iter-1", str(tmpdir.join("realization-*/iter-1")))
 
     ensset = EnsembleSet("reek001", [iter0, iter1])
+
+    tmpdir.chdir()
```

Comment:
Any reason to do `tmpdir.chdir()` twice in the same method? (also done on line 66) Would think that you are already in your tmpdir directory at this point? 
```suggestion
```

Reply:
Removed. Leftover line.